### PR TITLE
Feat: Fix Rtl Ui Buges For Server Badge

### PR DESCRIPTION
**Pull request by husamhammad**

### CODE REVIEW

File: server_badge/hooks.py [hunk 1]
```diff
@@ -9,7 +9,10 @@
 # ------------------
 
 # include js, css files in header of desk.html
-app_include_css = "/assets/server_badge/css/server.css"
+app_include_css = [
+    "/assets/server_badge/css/server.css",
+    "/assets/server_badge/css-rtl/rtl_style.css"
```
Comment (alaa-alsalehi) [inline server_badge/hooks.py:14]: why you do not follow the standered of rtl in erpnext?

Comment (alaa-alsalehi) [reply server_badge/hooks.py:14]: check what we are doing in Map PR

Comment (alaa-alsalehi) [reply server_badge/hooks.py:14]: https://github.com/frappe/frappe/pull/22248

**Comment by alaa-alsalehi** [comment]
@husamhammad It does not makes sense the last update does not show any css looks something wrong

**Comment by husamhammad** [comment]
The bundle on the CSS file will auto generate the CSS for RTL with all differences 
@alaa-alsalehi

**Pull request merged**